extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsTopologyManager.java

```diff
@@ -141,8 +142,14 @@ public KafkaStreams getStreams() {
 
     private void waitForTopicsToBeCreated(Set<String> topicsToAwait, String bootstrapServersConfig)
             throws InterruptedException {
-        Map<String, Object> config = new HashMap<>();
-        config.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServersConfig);
```

Comment:
Is it normal that we don't have this one anymore? (Note that I'm not familiar at all with the extension)

Reply:
Hi @gsmet, both `AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG` and `ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG` constants translate back to the exact same string `bootstrap.servers` property. Gunnar's suggestion was to use `AdminClientConfig` for this constant since it made more sense to use that since we are dealing with an AdminClient and not a consumer.

Reply:
OK, perfect, then.